assets/scss/_variables.scss

```diff
@@ -0,0 +1,37 @@
+// Background vars
+$bg-body            : #101116;
+$bg-content         : #161A25;
+$bg-button--primary : #242A36;
+
+// Color vars
+$color-icons            : #ACACAC;
+$color-primary          : #F97A2b;
+$color-secondary        : #F42C53;
+$color-light            : #FFFFFF;
+$color-light-secondary  : #A1A1A1;
+
+// Font related vars
+$font-family-base       : 'Montserrat', sans-serif;
+$font-family-headers    : 'Montserrat', sans-serif;
+
+$font-size-base         : 16px;
+$font-size-small        : ceil($font-size-base * 0.875); // 14px
+$font-size-large        : ceil($font-size-base * 1.125); //18px
+$font-size-h1           : ceil($font-size-base * 3.5); //56px
+$font-size-h2           : ceil($font-size-base * 2); // 32px
+$font-size-h3           : ceil($font-size-base * 1.5); // 24px
+$font-size-h4           : ceil($font-size-base * 1.3125); // 21px
+
+$font-weight-normal     : 400;
+$font-weight-medium     : 500;
+$font-weight-bold       : 700;
+
+// Grid breakpoints
+$break-sm               : 576px;
+$break-sm--min          : 575.98px;
+$break-md               : 768px;
+$break-md--min          : 767.98px;
+$break-lg               : 992px;
+$break-lg--min          : 991.98px;
+$break-xl               : 1200px;
+$break-xl--min          : 1199.98px;
```

Comment:
Can you please add a newline at the end of this file ?

EDIT: most files are missing the newline at the end.